README.md

```diff
@@ -113,9 +113,17 @@ Import Mandrill blacklist that you get from [here](https://mandrill.zendesk.com/
 
 If the list was successfully imported the CLI will return `OK`.
 
-### Import SendGrid Suppressions
+#### Import SendGrid Suppressions
```

Comment:
@ewandennis @yepher 
I think this simple instructions (we can fix copy as you guys suggest to make it even better) are enough. I think the new readme is too verbose and unnecessarily repetitive.